invenio_app_ils/circulation/api.py

```diff
@@ -260,9 +262,15 @@ def bulk_extend_loans(patron_pid, **kwargs):
     loans_search = get_all_range_expiring_ils_loans_by_patron_pid(
         expiring_in_days=days, patron_pid=patron_pid)
 
+    overdue_loans = get_all_overdue_loans() \
```

Comment:
The method `get_all_range_expiring_ils_loans_by_patron_pid` was introduced in the context of the bulk extension feature. Why not changing that method instead of performing 2 searching (more efficient)?
`get_all_range_expiring_ils_loans_by_patron_pid` -> `get_expiring_and_overdue_loans`